app/components/local-navbar.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import setCookie from '../helpers/setCookie';
+
+export default class ExampleComponent extends Component {
+  @action
+  changeTheme(event) {
+    const sunURL = 'assets/sun.png';
+    const moonURL = 'assets/moon.png';
+
+    if (event.target.src === 'https://dev.realdevsquad.com/assets/moon.png') {
```

Comment:
Could we create a constant for the link and point it here please

Reply:
I went for adding URL directly in src, because here, the URL will not change, but the image might get change. And even if we change the image, we will replace it with existing, so the URL will remain the same.

Reply:
1. Umm, you've set up dev.realdevsquad.com on your laptop/computer, but will the same link work on your phone?
2. Even according to your logic, if we tomorrow change the subdomain from `my.real......com` to `profile.real.....com`, will this URL be valid in that case?

Reply:
Ya. You are right. But when I logged event.target.src it showed `https://dev.realdevsquad.com/assets/moon.png` and if I just do `assets/moon.png` then it doesn't work.

Reply:
Yes. Public -> assets -> moon.png